.buildkite/deploy.sh

```diff
@@ -48,8 +48,14 @@ eval "$(ssh-agent)"
 ssh-add - <<< "${PLUTUS_CI_PRIVATE_SSH}"
 ssh-add -l
 
+# -----------------------------------------------------------------------------
```

Comment:
This comment seems slightly odd in that there's nothing after it and before the next bit?

Reply:
oops, that totally should not be there indeed!